express-notes-api/index.js

```diff
@@ -0,0 +1,103 @@
+const express = require('express');
+const app = express();
+const dataJson = require('./data.json');
+const notes = Object.values(dataJson.notes);
+const fs = require('fs');
+
+const jsonMiddleware = express.json();
+app.use(jsonMiddleware);
+
+app.get('/api/notes', (req, res) => {
+  res.json(notes);
+});
+
+app.get('/api/notes/:id', (req, res) => {
+  const noteId = parseInt(req.params.id, 10);
+  // eslint-disable-next-line no-console
+  console.log(notes);

```

Comment:
Please delete all console logs, they are only for debugging. You can leave in the `console.error` though.